reframe/core/pipeline.py

```diff
@@ -1120,7 +1121,10 @@ def check_performance(self):
 
             for val, reference in perf_values:
                 ref, low_thres, high_thres, *_ = reference
-                evaluate(assert_reference(val, ref, low_thres, high_thres))
+                try:
+                    evaluate(assert_reference(val, ref, low_thres, high_thres))
+                except SanityError as e:
+                    raise PerformanceError(e)
```

Comment:
```suggestion
                    raise PerformanceError() from e
```

Reply:
If I do it like that then the formatted message contains also the `:` from the `SanityError`.

Reply:
Can you copy here the output you get?

Reply:
`Reason: performance error: : 10 is beyond reference value 20 (l=18.0, u=22.0)`

Reply:
I see, but `PerformanceError(e)` is not the way to go. We don't do it anywhere else in the framework. You may improve the way that exception messages are chained, by not printing the `:` the current exception's message is empty.